src/raiden_libs/service_registry.py

```diff
@@ -117,14 +118,15 @@ def checked_transact(
         f"\n\tSee {etherscan_url_for_txhash(web3.eth.chain_id, transaction_hash)}"
     )
 
-    transaction_receipt = web3.eth.wait_for_transaction_receipt(transaction_hash, poll_latency=1.0)
+    timeout = 60 * 10  # 10mins
+    transaction_receipt = web3.eth.wait_for_transaction_receipt(transaction_hash, timeout, 1.0)
     if wait_confirmation_interval:
         while (
             "blockNumber" not in transaction_receipt
             or web3.eth.block_number
             < transaction_receipt["blockNumber"] + DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS
         ):
-            transaction_receipt = web3.eth.wait_for_transaction_receipt(transaction_hash)
+            transaction_receipt = web3.eth.wait_for_transaction_receipt(transaction_hash, timeout)
```

Comment:
The default is `poll_latency=0.1` - I assume this is the poll-interval (not documented in web3). Why are we polling 10x more often when waiting for the confirmation than for the initial block inclusion?

Reply:
I just noticed that this was there before - but maybe you know the reason anyways :)

Reply:
No, actually no clue, but I can set it to 1.0 as well.